src/main/java/press/lis/greeb/SampleJobService.java

```diff
@@ -20,9 +20,9 @@
     private Bot bot;
 
 
-    @Scheduled(cron = "*/5 * * * * *")
+    @Scheduled(cron = "*/5 * * * * *") // sec, min, hr, day, month, weekday
```

Comment:
Generally, I wouldn't have left this comment, because for me that's a part of common knowledge and should be checked through better knowledge source.

Despite that, it should be as short as possible.